packages/common/source/types/index.ts

```diff
@@ -299,11 +299,16 @@ export interface ExternalActivityLog {
 
 export type ActivityLogPage = ExternalListPage<ExternalActivityLog>;
 
+interface Object {
```

Comment:
Rename this to `ManifestValues`. `Object` is already defined in the global scope.